ui/components/TracePage/TracePage.tsx

```diff
@@ -478,7 +498,7 @@ const Call = ({ id, refreshArcherArrows }: { id: string; refreshArcherArrows: ()
             )}
           </ArcherElement>
           <div className="mx-2">
-            <CallName className="text-base text-slate-700" id={id} />
+            <CallName className="text-base text-slate-700" {...{ cls, name }} />
```

Comment:
Spreading props disables type-checking (sigh) and seems unnecessary here, let's avoid it

Reply:
yeah, several aspects of TSX typechecking leave something to be desired and I'm not sure whose fault it is 😞